src/NetCoreInstaller/Targets/packaging/windows/NetCoreInstaller/variables.wxi

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <?define Manufacturer = Microsoft Corporation?>
+
+    <?define InstallerDataRootFolder=[CommonAppDataFolder]Microsoft .NET Core?>
+    <?define InstallLocationVariable=InstallDir?>
```

Comment:
You can probably take this out since we're not using the custom install location UI

Reply:
So take this out as well?
`InstallLocationVariable="$(var.InstallLocationVariable)"`

Reply:
Done